api/src/services/users/users.js

```diff
@@ -6,3 +6,45 @@ export const getCustomerId = async ({ id }) => {
     where: { id: parseInt(id) },
   })
 }
+
+export const getUserByCustomerId = async ({ customerId }) => {
+  return await db.user.findUnique({
```

Comment:
You have to await an async function anyway, and since you're not doing anything with `db.user.findUnique`'s return, you can just make it a regular function that returns a promise
```suggestion
export const getUserByCustomerId = ({ customerId }) => {
  return db.user.findUnique({
```

Reply:
@jtoar thanks for that! :)